roles/sap_general_preconfigure/meta/argument_specs.yml

```diff
@@ -341,3 +341,11 @@ argument_specs:
           sap_general_preconfigure_db_group_name: 'dba'
         required: false
         type: str
+
+      sap_general_preconfigure_default_locale:
+        description:
+          - Use this variable to specify the default system locale.
+        example:
+          sap_general_preconfigure_db_group_name: 'en_US.UTF-8'
```

Comment:
I am guessing you mean:
sap_general_preconfigure_default_locale: 'en_US.UTF-8' -- or C.UTF-8 depending what you think should be the "default" recommendation :)

Reply:
Thanks @Klaas- fixed this one, in addition @berndfinger and I found a bug in the algorithm so that this was variable was never used in certain circumstances (non-English locale set as default would always stop although variable was set)
This variable above is just an example, per default nothing is changed (variable empty in defaults/main.yml)

Reply:
I also removed the code from being executed on SLES because the reviewers are not available for the next weeks -- We may need to open another PR to enable this on SLES